examples/field_formats_example/public/examples/1_using_existing_format.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { SerializedFieldFormat } from '../../../../src/plugins/field_formats/common';
+import { FieldFormatsStart } from '../../../../src/plugins/field_formats/public';
+
+// 1. Assume we have an existing field format configuration serialized and saved somewhere
+// In this case it is `bytes` field formatter with a configured `'0.00b'` pattern
+export const sampleSerializedFieldFormat: SerializedFieldFormat<{ pattern: string }> = {
```

Comment:
Outside the scope of this PR but worth noting - we should have a type specific to this formatter.

Reply:
Yes! 100% agree. I created https://github.com/elastic/kibana/issues/108158#issuecomment-896654411